assemblies/assembly_openshift-authoring-managed/main.adoc

```diff
@@ -35,9 +35,15 @@ and other business assets.
 
 [NOTE]
 ====
-Since {PRODUCT} version 7.5, support for {OPENSHIFT} 3.x is deprecated, including installation using all templates and using the Automation Broker (Ansible Playbook). New features might not be added, and this functionality will be removed in a future release. 
+Since {PRODUCT} version 7.5, support for {OPENSHIFT} 3.x is deprecated, including installation using all templates. New features might not be added, and this functionality will be removed in a future release. 
 ====
 
+[NOTE]
+====
+Templates are not supported for use with {OPENSHIFT} 4.x. To deploy {PRODUCT} on {OPENSHIFT} 4.x, see the instructions in {URL_DEPLOYING_OPENSHIFT_OPERATOR}[_{DEPLOYING_OPENSHIFT_OPERATOR}_].
```

Comment:
This would be factually wrong, OpenShift 4 does support templates in general, it's just that Red Hat does not support using these particular templates with OCP 4. (And maybe they won't work at allm but some templates do work)

Reply:
You should say something like "You cannot use templates with OS4. To deploy...."

Reply:
What is the customer experience here? We need to highlight that. If the user cannot use templates in OpenShift 4, we need to state that clearly and not beat about the bush.

Reply:
I asked and they don't know because they did not try. Putting in a "do not use".